packages/terra-hyperlink/tests/wdio/hyperlink-spec.js

```diff
@@ -0,0 +1,120 @@
+/* global browser, Terra, before */
+const viewports = Terra.viewports('tiny', 'medium', 'large');
+
+describe('Hyperlink', () => {
```

Comment:
nit: we can drop all references to { viewports } in this test file once simplify to `tiny` using
```
before(() => browser.setViewportSize(Terra.viewports('tiny')[0]));
```

Reply:
Updated: https://github.com/cerner/terra-core/commit/5d1a6de0b4dd80fbbb620354ee5b823764386a5b